### PR TITLE
Updated Seed File and Load Testing

### PR DESCRIPTION
**Pull request by adauzat90**

### CODE REVIEW

File: zlp-scheduler/app/controllers/concerns/scheduler_2.rb [hunk 1]
```diff
@@ -74,7 +75,7 @@ def self.Generate_time_slots(cohort)
                             if @conflict.is_a? false.class
                                 break
                             else
-                                #print(@conflict)
+                                print(@conflict)
```
Comment (BWitemeyer) [inline zlp-scheduler/app/controllers/concerns/scheduler_2.rb:78]: Can you re-comment out this line?

File: zlp-scheduler/app/controllers/admin_controller.rb [hunk 2]
```diff
@@ -156,8 +156,8 @@ def view_result
     @cohort = Cohort.find(params[:cohort_id])
     # Scheduler_2.Generate_time_slots(@cohort)
     @date_dict = { "M" => "Monday", "T" => "Tuesday", "W" => "Wednesday", "TR" => "Thursday", "F" => "Friday"}
-    @results = TimeSlot.where(:was_conflict => false).order(cost: :asc,id: :asc).limit(10)
-    @conflicts = TimeSlot.where(:was_conflict => true).order(cost: :asc,id: :asc).limit(10)
+    @results = TimeSlot.where(:was_conflict => false).order(cost: :asc,id: :asc).limit(200)
+    @conflicts = TimeSlot.where(:was_conflict => true).order(cost: :asc,id: :asc).limit(100)
```
Comment (BWitemeyer) [inline zlp-scheduler/app/controllers/admin_controller.rb:160]: I just realized that this is not filtering based on `cohort_id`. Could we add that so multiple cohorts can be supported?

Comment (adauzat90) [reply zlp-scheduler/app/controllers/admin_controller.rb]: Good Catch! it should be fixed now

**Review by BWitemeyer — APPROVED** [review]
Looks good!

**Pull request merged**